packages/ui/src/uiManager.ts

```diff
@@ -57,4 +61,46 @@ export class UIManager {
         this._options.bannerManager && this._options.bannerManager.reset();
         this._options.toastManager && this._options.toastManager.reset();
     }
+
+    private _loadFont(): void {
+        let fontFamily = this._options.corePlayer.config.contrib.ui.fonts.fontFamily;
```

Comment:
please wrap this method logic with:
1. try... catch.... - write a noticeable `warning` indicating that font overrides failed.
2. add a boolean flag to make sure second execution is ignored
3. consider changing the name to `overrideCorePlayerStyles` (since it doesn't actually load fonts)

Reply:
what do you say about moving all fonts / style overriding to a different manager and provide it   to the uiManger in its' options ?

Reply:
done